src/components/modules/dashboard/offer/ui/dialogs/LoanRequestDetailDialog.tsx

```
@@ -0,0 +1,147 @@
+              {formatDateFromFirebase(
+                selectedRequest.createdAt?.seconds,
+                selectedRequest.createdAt?.nanoseconds,
+              )}
+            </p>
+          </div>
+        </div>
+
+        <div className="mt-6">
+          <p className="font-semibold mb-2">Milestones</p>
+          <ul className="space-y-2">
+            {selectedRequest.milestones?.map((m: any, idx: number) => (
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)